database/relresolver.go

```diff
@@ -1,17 +1,31 @@
-package mysql
+package database
 
 import (
 	"errors"
+	"fmt"
 
 	"github.com/spf13/viper"
 )
 
+// RelationshipResolver ...
+type RelationshipResolver struct {
+	g ConfigReader
+}
+
 // ConfigReader ...
 type ConfigReader struct {
 	v *viper.Viper
 }
 
-func (g *ConfigReader) readPrimaryRecord() (pRecordType string, err error) {
+// NewConfigReader returns an initialised instance of ConfigReader
+func NewConfigReader(v *viper.Viper) *ConfigReader {
+	return &ConfigReader{
+		v,
+	}
+}
+
+// ReadPrimaryRecord ...
+func (g *ConfigReader) ReadPrimaryRecord() (pRecordType string, err error) {
```

Comment:
I think this can be private, it's only used in the same package

Reply:
Done

Reply:
Turns out [`dumper`](https://github.com/hellofresh/klepto/blob/patch/dump-configed-record-only/database/mysql/dumper.go#L135), which is in the `mysql` package also uses it.